introduction.md

```diff
@@ -1,2 +1,7 @@
 Hi, I am Ivan and I am learning about GitHub.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?